packages/babel-core/src/errors/rewrite-stack-trace.ts

```diff
@@ -70,6 +70,18 @@ function CallSite(filename: string): CallSite {
   } as CallSite);
 }
 
+// https://github.com/nodejs/node/pull/38215/files#diff-670bf55805b781d9a3579f8bca9104c04d94af87cc33220149fd7d37b095ca1cR210
+function isErrorStackTraceLimitWritable() {
+  const desc = Object.getOwnPropertyDescriptor(Error, "stackTraceLimit");
+  if (desc === undefined) {
```

Comment:
We first check if the property exists, so the descriptor will never be undefined when we call this function.

Reply:
You mean `Error.stackTraceLimit &&= Math.max(
     Error.stackTraceLimit,
     MIN_STACK_TRACE_LIMIT,
   );`?
If we can't increase `Error.stackTraceLimit`, maybe there is no need to enable this feature? (Just a guess, I'm not sure if it would be better to enable it with only the default 10 stack frames)

Reply:
I got confused by the `!!Error.captureStackTrace` check above, I thought it was `!!Error.stackTraceLimit`